client/js/libs/handlebars/localetime.js

```diff
@@ -1,7 +1,17 @@
 "use strict";
 
 const moment = require("moment");
+const constants = require("../../constants");
 
 module.exports = function(time) {
-	return moment(time).format("D MMMM YYYY, HH:mm:ss");
+	const options = require("../../options");
```

Comment:
Why is this part of the function itself? To make sure it uses the most up-to-date values set by user?